programs/bpf/Cargo.lock

```diff
@@ -161,16 +161,16 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bumpalo"
-version = "3.1.2"
+version = "3.2.1"
```

Comment:
Casually audited this and found no problem fixing the security hole: https://github.com/fitzgen/bumpalo/blob/master/CHANGELOG.md#321